modules/library/main/src/test/java/org/geotools/data/util/MeasureConverterTest.java

```diff
@@ -28,13 +28,13 @@
 public class MeasureConverterTest {
 
     @Test
-    public void testToMeasure() {
+    public void testStringToMeasure() {
         assertEquals(new Measure(10, SI.METRE), Converters.convert("10m", Measure.class));
         assertEquals(new Measure(0.3, USCustomary.FOOT), Converters.convert(".3ft", Measure.class));
+        assertEquals(new Measure(3, null), Converters.convert("3", Measure.class));
         assertEquals(
                 new Measure(3e-10, NonSI.DEGREE_ANGLE),
                 Converters.convert("3e-10\u00B0", Measure.class));
-        assertEquals(new Measure(3, null), Converters.convert("3", Measure.class));
```

Comment:
Why is this test gone?

Reply:
Only the order changed, but this also appears to be unintentional.